### ISSUE TITLE
Invalid symlinks in `git/checkouts` cause `cargo-trim` to quit, with a cryptic message.

### ISSUE THREAD

**Issue by sasanj** [iamsauravsharma/cargo-trim#60]
Hi.
There was an *invalid* `symlink` in one of the projects in the `git/checkouts`:

``` bash
> la $CARGO_HOME/git/checkouts/surfman-a5c700867c3db833/fb78226/android-example/app/src/main/jniLibs/arm64-v8a/

lrwxrwxrwx 82 10 Mar  2020 libsurfman_android_threads.so -> ../../../../../../target/aarch64-linux-android/debu/libsurfman_android_threads.so

```
 This caused `cargo-trim` refusing to work:

``` bash
> cargo trim list -a
Error: failed to get folder size

Caused by:
    No such file or directory (os error 2)
```
Of course removing the `symlink`, solved the problem.
As an after thought, I don't think there would any need to check for a `symlink` size.

**Comment by iamsauravsharma** [comment]
#61 fixed issue related to symlink

**Comment by iamsauravsharma** [comment]
Released v0.10.2 with above fix
